sdk/formrecognizer/Azure.AI.FormRecognizer/tests/samples/Sample_CreateComposedModelAsync.cs

```diff
@@ -0,0 +1,105 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using Azure.AI.FormRecognizer.DocumentAnalysis.Tests;
+using Azure.Core.TestFramework;
+using NUnit.Framework;
+
+namespace Azure.AI.FormRecognizer.DocumentAnalysis.Samples
+{
+    public partial class DocumentAnalysisSamples : SamplesBase<DocumentAnalysisTestEnvironment>
+    {
+        [Test]
+        public async Task CreateComposedModelAsync()
+        {
+            string endpoint = TestEnvironment.Endpoint;
+            string apiKey = TestEnvironment.ApiKey;
+            string trainingFileUrl = TestEnvironment.BlobContainerSasUrl;
```

Comment:
nit: should we suffix these two with `Uri` for consistency?

Reply:
at this point, both variables contain a raw `URL` in the form of a string, so I prefer to leave it as `Url`.
when the variable is a `Uri` we do use `Uri` in the variable name. if you don't see that, let me know